pkg/services/store/entity/migrations/entity_store_mig.go

```diff
@@ -120,7 +120,11 @@ func initEntityTables(mg *migrator.Migrator) string {
 		},
 		Indices: []*migrator.Index{
 			{Cols: []string{"guid", "resource_version"}, Type: migrator.UniqueIndex},
-			{Cols: []string{"namespace", "group", "resource", "name", "resource_version"}, Type: migrator.UniqueIndex},
+			{
+				Cols: []string{"namespace", "group", "resource", "name", "resource_version"},
```

Comment:
be sure the change the "marker" above so this migration happens on existing setups.  (yes it will wipe out everything first)